src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -15,8 +15,7 @@
 namespace Microsoft.AspNetCore.Authentication.Cookies
 {
     public class CookieAuthenticationHandler :
-        AuthenticationHandler<CookieAuthenticationOptions>,
-        IAuthenticationSignInHandler,
+        SignInAuthenticationHandler<CookieAuthenticationOptions>,
         IAuthenticationSignOutHandler
```

Comment:
Do you still need IAuthenticationSignOutHandler? Isn't it transitive?

Reply:
Yeah nope, good catch don't need it anymore